### ISSUE TITLE
'apply' button is broken

### ISSUE THREAD

**Issue by fpgeek** [mesosphere/marathon#3903]
I found that 'apply' button is broken on the marathon v1.1.1.
<img width="770" alt="2016-05-17 4 36 10" src="https://cloud.githubusercontent.com/assets/1998160/15315665/068a865e-1c54-11e6-9673-13923f6f83d4.png">

It's because 'appVersions' property of 'AppVersionsStore' object hasn't the latest version app.
```
// Diff the new configuration against the current app settings
  getAppConfigDiff: function (appId, newConfig) {
    var allVersions = this.getAppVersions(appId);
    if (allVersions.length === 0) {
      return newConfig;
    }
    var latestVersion = allVersions[0];
    return AppConfigTransforms.diff(this.appVersions[latestVersion], newConfig);
  }
```

https://github.com/mesosphere/marathon-ui/pull/766

**Comment by gkleiman** [comment]
cc/ @pierlo-upitup

**Comment by orlandohohmeier** [comment]
Fixed in mesosphere/marathon-ui#766
